src/main/java/org/bladerunnerjs/contrib/contentplugin/appcache/AppcacheManifestBuilder.java

```diff
@@ -52,16 +45,11 @@ public AppcacheManifestBuilder(BRJS brjs, BundleSet bundleSet, String brjsVersio
 	 */
 	public String getManifest(RequestMode requestMode) throws PropertiesException, ContentProcessingException, ConfigException, MalformedTokenException
 	{
-		String manifest = getManifestHeader();
-        if(version != null)
```

Comment:
`version` will never be null at this point as it's now set by the central BRJS version. Each time the app cache is generated it will contain *a* version. The default version in 'dev' is 'dev' (this can be overridden), and the default during `build-app` is a timestamp (again this can be overridden).
I've tested this with an app and it all seems to work as it did before.